Kahve BOT/dist/metadata.json

```diff
@@ -0,0 +1,21 @@
+{
+    "author": {
+        "name": "Eren",
+        "id": "478937019758673931"
+    },
+    "service": "Kahve",

```

Comment:
Service name must be the same as the folder name.

Reply:
Eh no

Reply:
Was that not needed anymore?

Reply:
I thought it was still needed due to the source code button on the presence page.

Reply:
I mean try out and tell @Voknehzyr if its still broken